scripts/fetch-archive.js

```diff
@@ -33,31 +31,35 @@ function authenticate() {
   })
 }
 
-async function getProjectGitHubData(repo) {
-  const [ owner, repoName ] = repo.split('/')
+async function getProjectGitHubData (repo) {
+  const [owner, repoName] = repo.split('/')
   const { data } = await octokit.repos.get({ owner, repo: repoName })
   const { stargazers_count, forks_count, open_issues_count } = data
   return { stars: stargazers_count, forks: forks_count, issues: open_issues_count }
 }
 
-async function getAllProjectGitHubData(repos) {
+async function getAllProjectGitHubData (repos) {
   const data = []
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
I left the following lines untouched and added `// eslint-disable-next-line` to suppress the following errors:
```
   46:3   error  iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations  no-restricted-syntax
   47:5   error  Unexpected `await` inside a loop                                                                                                                                           no-await-in-loop
   49:24  error  Unexpected `await` inside a loop                                                                                                                                           no-await-in-loop
```